e2e/diff/no_schema_diff_test.go

```diff
@@ -0,0 +1,98 @@
+package e2e
+
+import (
+	"context"
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	"ariga.io/atlas/sql/migrate"
+	"entgo.io/ent/dialect/sql/schema"
+
+	entMigrate "github.com/openmeterio/openmeter/internal/ent/db/migrate"
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+)
+
+var PG_URL string
+
+func TestMain(m *testing.M) {
+	PG_URL = os.Getenv("POSTGRES_URL")
+
+	os.Exit(m.Run())
+}
+
+const (
+	DIR = "./tmp/"
+)
+
+func setup(t *testing.T) {
+	if PG_URL == "" {
+		t.Skip("POSTGRES_URL not set")
+	}
+}
+
+func TestNoSchemaDiffOnMigrate(t *testing.T) {
+	setup(t)
+	driver, err := entutils.GetPGDriver(PG_URL)
```

Comment:
You could merge these into a single function (if reusability is needed). You can drop the global variable that way.

I'd also use `t.Helper()` to make sure the correct line is reported in case of an error.